### ISSUE TITLE
Naming conflict with "vim-bbye"

### ISSUE THREAD

**Issue by GeneZharov** [Asheq/close-buffers.vim#10]
After the recent change in command namings, I get a conflict with [vim-bbye](https://github.com/moll/vim-bbye).

```
Error detected while processing /home/user/.config/nvim/plugged/close-buffers.vim/plugin/close-buffers.vim:
line   33:
close-buffers.vim: You already have a ":Bdelete" command defined
line   39:
close-buffers.vim: You already have a ":Bwipeout" command defined
```

**Comment by Asheq** [comment]
Thanks for the report. In the short term, I'll allow you to specify a different command name of your choosing to avoid naming conflicts, like this:

```
let g:close_buffers_bdelete_command = 'Bclose'
let g:close_buffers_bwipeout_command = 'Bwipeout'
```

In the long run, I plan on incorporating the features of vim-bbye into this plugin so you will only need this one.

Not sure exactly how yet but right now I'm thinking of adding a `-keepwindows` flag like this:

```
:Bdelete hidden -keepwindows
```

**Comment by Asheq** [comment]
Closed by #16
